src/HomeAssistant.h

```diff
@@ -74,13 +74,19 @@ class FuFarmHomeAssistant
   /**
    * Publishes MQTT messages for configured sensors.
    * In some cases, if a sensor value is the same as the previous one the MQTT message won't be published.
-   * It means that MQTT messages will produced each time the setValues method is called but they might not
+   * It means that MQTT messages will produced each time the update method is called but they might not
```

Comment:
There's a typo, but it's probably better stated as:
`This means MQTT messages are not always produced when the update method is called.`